classroom.rb

```diff
@@ -0,0 +1,15 @@
+require './student'
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_students(student)
+    @students.push(@student)
+    student.classroom = self
```

Comment:
- [ ] <b>REQUIRED:</b> There is a mistake here. online 12, the value you are pushing in `@students` is the array value declared on line 8 not the value from the method's parameter.  and the other thing,  we don't need to initialize `@students`(As you did on line 8)  since it is already declared on line 4.